e2e-tests/cypress/support/insurance/account/complete-and-submit-password-reset-form-maximum-retries.js

```diff
@@ -0,0 +1,24 @@
+import { ACCOUNT } from '../../../../constants/account';
+import completeAndSubmitPasswordResetForm from './complete-and-submit-password-reset-form';
+import { backLink } from '../../../e2e/pages/shared';
+
+const attemptsToMake = [...Array(ACCOUNT.MAX_PASSWORD_RESET_TRIES)];
+
+/**
+ * completeAndSubmitPasswordResetFormMaximumRetries
+ * Complete and submit the password reset form multiple times,
+ * Mathcing the maximum retries allowed before an account becomes blocked.
+ */
+const completeAndSubmitPasswordResetFormMaximumRetries = () => {
+  attemptsToMake.forEach((item, index) => {
```

Comment:
would a for loop make more sense here?

Reply:
I don't see a benefit to using a for loop instead of forEach? We're also using forEach throughout the E2E suite, no other instances of for loops